modules/standard/Random.chpl

```diff
@@ -638,6 +638,23 @@ module Random {
         const oddseed = if seed % 2 == 0 then seed + 1 else seed;
         return oddseed;
       }
+      /*
+      Generate a seed based on the entropy pool in the OS; this requires
+      the presence of /dev/urandom.  This should be standard on both macOS
+      and Linux systems.
+
+      For highly parallel applications on one locale, it may be necessary
+      to lock this object to ensure two tasks do not both pull the same
+      bit of entropy.
+      */
+      proc type udevRandomSeed: int(64) {
```

Comment:
Does this name originate from any existing libraries out in the wild? Unless there is precedent for this name, I might suggest something more concise, like `devRandom`

Reply:
OH.  Wow, yeah, no, I put the u in the wrong place.  That should be devURandomSeed, if anything, to differentiate it from /dev/random.

Reply:
I might rather use a different name like "kernel random" or something like that. I don't think that we're reading it from `/dev/urandom` matters that much.

Reply:
Kernel random is a great idea.